lib/command/config.rb

```diff
@@ -103,6 +103,11 @@ def disable_dsym?
       @dsl_config[:disable_dsym]
     end
 
+    def still_download_sources?(name = nil)
```

Comment:
should we: `return false unless name`

Reply:
Oh, I forgot to remove the default param. It's supposed to be non-nil